GameData/Kerbalism/System/Misc.cfg

```diff
@@ -83,16 +83,51 @@
 
 
 // ============================================================================
-// Set part tags
+// Set generic part tags
 // ============================================================================
 
-@PART[kerbalism-*]:FOR[Kerbalism] { %tags = _kerbalism }
-@PART[kerbalism-greenhouse,kerbalism-container-inline-*,kerbalism-lifesupportmodule]:FOR[Kerbalism]
+@PART[kerbalism-*]:FOR[zzzKerbalism] { %tags = _kerbalism }
+@PART[kerbalism-greenhouse,kerbalism-container-inline-*,kerbalism-lifesupportmodule]:FOR[zzzKerbalism]
  { %tags = #$tags$ life support }
 
 
+// ============================================================================
+// Set part specific tags
+// ============================================================================
+
+@PART[kerbalism-activeshield]:AFTER[zzzKerbalism] { %tags = #$tags$ magnetic field radiation shield }
+@PART[kerbalism-chemicalplant]:AFTER[zzzKerbalism] { %tags = #$tags$ external chemical plant electrolysis o2 oxygen h2 hydrogen h2o water sabatier haber liquidfuel nh3 ammonia waste incinerator compressor anthraquinone hydrazine mre soe sco }
+@PART[kerbalism-lifesupportmodule]:AFTER[zzzKerbalism] { %tags = #$tags$ external eclss scrubber pressure control atmosphere wasteatmosphere o2 oxygen nitrogen co2 carbondioxide humidity h2o water recycler waste processor fuelcell fuel cell monoprop }
+@PART[kerbalism-geigercounter]:AFTER[zzzKerbalism] { %tags = #$tags$ geiger counter science sensor }
+@PART[kerbalism-gravityring]:AFTER[zzzKerbalism] { %tags = #$tags$ habitat crew gravity rotate comfort }
+@PART[kerbalism-greenhouse]:AFTER[zzzKerbalism] { %tags = #$tags$ habitat crew greenhouse food crop hydroponic co2 carbondioxide waste atmosphere wasteatmosphere h2o water o2 oxygen }
+@PART[kerbalism-container-inline-*]:AFTER[zzzKerbalism] { %tags = #$tags$ supply supplies food water waste wastewater container }
+@PART[kerbalism-container-inline-0625]:AFTER[zzzKerbalism] { %tags = #$tags$ 625 0625 0.625 .625 }
+@PART[kerbalism-container-inline-125]:AFTER[zzzKerbalism] { %tags = #$tags$ 125 1.25 }
+@PART[kerbalism-container-inline-250]:AFTER[zzzKerbalism] { %tags = #$tags$ 250 2.5 2.50 }
+@PART[kerbalism-container-inline-375]:AFTER[zzzKerbalism] { %tags = #$tags$ 375 3.75 }
+@PART[kerbalism-*pump]:AFTER[zzzKerbalism] { %tags = #$tags$ harvest extract pump h2o water nitrogen nh3 ammonia }
+@PART[kerbalism-container-radial-*]:AFTER[zzzKerbalism] { %tags = #$tags$ external radial tank pressurized o2 oxygen nitrogen h2 hydrogen nh3 ammonia co2 carbondioxide }
+@PART[kerbalism-antenna]:AFTER[zzzKerbalism] { %tags = #$tags$ antenna }
+
```

Comment:
use FOR[Kerbalism] on these ;)

Reply:
Hmm. Line 89? That would overwrite all of them.

Reply:
Good change, but not really under "Increase-all-crewable-parts'-EC-because-ECLSS-uses-EC-constantly", is it?

Reply:
I could open a dozen branches.
Should I?
Confuse me and everybody else as well?

However, I'm new to this.